src/docs/markdown/caddyfile/options.md

```diff
@@ -214,9 +214,21 @@ The differs from the [`log` directive](/docs/caddyfile/directives/log), which on
 - **output** configures where to write the logs. See the [`log` directive](/docs/caddyfile/directives/log#output-modules) for complete documentation.
 - **format** describes how to encode, or format, the logs. See the [`log` directive](/docs/caddyfile/directives/log#format-modules) for complete documentation.
 - **level** is the minimum entry level to log. Default: `INFO`.
-- **include** specifies the log names to be included in this logger. For example, to include only logs emitted by the admin API, you would include `admin.api`. By default, all logs are included.
-- **exclude** specifies the log names to be excluded from this logger. For example, to exclude only HTTP access logs, you would exclude `http.log.access`. By default, no logs are excluded.
+- **include** specifies the log names to be included in this logger. For example, to include only logs emitted by the admin API, you would include `admin.api`. By default, all logs are included (aka this list is empty by default).
+- **exclude** specifies the log names to be excluded from this logger. For example, to exclude only HTTP access logs, you would exclude `http.log.access`. By default, no logs are excluded (aka this list is empty by default).
 
+The logger names that `include` and `exclude` accept depend on the modules used, and the easiest way to find them out is from the logs.
+
+Here is an example logging as json all http access logs and admin logs to stdout, but not http error logs:
+
+```caddy
+log default {
+    output stdout
+	format json
+	include http.log.access admin.api
+	exclude http.log.error
+}
```

Comment:
This example doesn't make much sense, since `include` and `exclude` are seldom used together in practice, and won't serve any purpose in this case: the log will only ever include `http.log.access` and `admin.api` logs, so excluding `http.log.error` is nonsensical.

Maybe if `http.log.access` was changed to `http.log` that would make more sense, but it's still shorter to write the config by omitting `exclude`, since I think the only loggers under `http.log` are `.access` and `.error` anyways.